### PR TITLE
Update govuk-notify doc

### PR DESCRIPTION
**Pull request by 1pretz1**
Includes information on Notify's daily email sending limit with Amazon
SES.

### CODE REVIEW

File: source/manual/govuk-notify.html.md [hunk 1]
```diff
@@ -24,16 +24,17 @@ for each environment):
...
-  of 350 requests per second meaning we should adhere to this limit when using
-  Notify to deliver emails for our applications. This is set in email-alert-api
-  and can be found in the [DeliveryRequestWorker][DeliveryRequestWorker]. If you
-  wish to double check this figure you could ask in their slack channel
-  #govuk-notify.
+  **Note:** currently (as of 04/08/2020) GOV.UK Notify has a maximum rate limit
+  of 350 requests per second and a daily limit of 30 million emails per day
+  (Notify's daily allowance with Amazon SES) meaning we should adhere to these
+  limits when using Notify to deliver emails for our applications. The rate
+  limit is set in email-alert-api and can be found in the
```
Comment (benthorner) [inline source/manual/govuk-notify.html.md:35]:
```suggestion
  per second limit is set in email-alert-api and can be found in the
```

File: source/manual/govuk-notify.html.md [hunk 2]
```diff
@@ -24,16 +24,17 @@ for each environment):
...
-  of 350 requests per second meaning we should adhere to this limit when using
-  Notify to deliver emails for our applications. This is set in email-alert-api
-  and can be found in the [DeliveryRequestWorker][DeliveryRequestWorker]. If you
-  wish to double check this figure you could ask in their slack channel
-  #govuk-notify.
+  **Note:** currently (as of 04/08/2020) GOV.UK Notify has a maximum rate limit
+  of 350 requests per second and a daily limit of 30 million emails per day
+  (Notify's daily allowance with Amazon SES) meaning we should adhere to these
+  limits when using Notify to deliver emails for our applications. The rate per
+  second limit is set in email-alert-api and can be found in the
```
Comment (benthorner) [inline source/manual/govuk-notify.html.md:35]:
```suggestion
  per second limit is set in email-alert-api and can be found in the
```

The "second" limit would be the daily one in the previous sentence 😸

Comment (1pretz1) [reply source/manual/govuk-notify.html.md]: I've removed all ambiguity 😄

File: source/manual/govuk-notify.html.md [hunk 3]
```diff
@@ -24,16 +24,17 @@ for each environment):
...
-  of 350 requests per second meaning we should adhere to this limit when using
-  Notify to deliver emails for our applications. This is set in email-alert-api
-  and can be found in the [DeliveryRequestWorker][DeliveryRequestWorker]. If you
-  wish to double check this figure you could ask in their slack channel
-  #govuk-notify.
+  **Note:** currently (as of 04/08/2020) GOV.UK Notify has a maximum rate limit
+  of 350 requests per second and a daily limit of 30 million emails per day
+  (Notify's daily allowance with Amazon SES) meaning we should adhere to these
+  limits when using Notify to deliver emails for our applications. The rate
+  limit of 350 requests per second is set in email-alert-api and can be found in the
```
Comment (benthorner) [inline source/manual/govuk-notify.html.md:35]:
```suggestion
  limit of 350 requests per second is set in email-alert-api and can be found in the
```
```suggestion
  limit of 350 requests per second, for sending emails to Notify, is set in email-alert-api and can be found in the
```

🤣

Comment (1pretz1) [reply source/manual/govuk-notify.html.md:35]: Ah ha, we aren't actually sending emails to Notify, only mere requests... 

I'm merging and running away from this PR! 😆

**Pull request merged**